src/getDependencyConfig.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Given an array of dependencies - it returns their RNPM config
+ * if they were valid.
+ */
+module.exports = function getDependencyConfig(config, deps) {
```

Comment:
@Kureev not a huge fan of how this function is named, but couldn't think of anything better. Feel free to update it!